components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -80,6 +80,7 @@
 
     <app-create-object-modal
       [visible]="createModalVisible"
+      [projectNoteVisible]="true"
```

Comment:
I confess that a better name is not occurring to me atm, but I'm trying to think of a clearer variable name than `projectNote` ...

Reply:
No worries; I deleted that variable as it is not actually needed.